pkg/diagnose/connectivity/core_endpoint.go

```diff
@@ -198,9 +198,16 @@ func verifyEndpointResponse(statusCode int, responseBody []byte, err error) (str
 
 	var verifyReport string
 	var newErr error
+
+	scrubbedResponseBody := scrubber.ScrubLine(string(responseBody))
+	if !diagCfg.Verbose && len(scrubbedResponseBody) > 500 {
+		scrubbedResponseBody = fmt.Sprint(len(scrubbedResponseBody) > 500, scrubbedResponseBody[:500]+"...")
```

Comment:
This will print `true blablabla...`, I guess this is not what you intended ?
```suggestion
		scrubbedResponseBody = scrubbedResponseBody[:500]+"..."
```

We could add a message at the end to mention the full size but definitely not required, eg. `[full body is 550 bytes, truncated at 500]`

Reply:
HOOOO good one, that's a miss one when I was testing in local I'll fix that
good one for the print to I'll add it